bakerydemo/recipes/blocks.py

```diff
@@ -29,6 +29,13 @@ class Meta:
         icon = "tick"
 
 
+class CustomTypedTableBlock(TypedTableBlock):
+    def get_preview_value(self):
+        # Avoid having to construct TypedTable ourselves. This won't be
+        # necessary if https://github.com/wagtail/wagtail/pull/12808 is merged
+        return self.to_python(self.meta.preview_value)
```

Comment:
Ref: https://github.com/wagtail/wagtail/pull/12808

Reply:
Yes, but it's currently still in `main`, so this is still necessary for stable 6.4.x! 😬